cmd/configschema/go.mod

```diff
@@ -802,3 +803,5 @@ exclude github.com/StackExchange/wmi v1.2.0
 
 // see https://github.com/distribution/distribution/issues/3590
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace go.opentelemetry.io/collector/pdata => go.opentelemetry.io/collector/pdata v0.0.0-20220412005140-8eb68f40028d
```

Comment:
I don't think we need this, this replaces the same version used above

Reply:
Some modules depend on go.opentelemetry.io/collector that has go.opentelemetry.io/collector/pdata@v0.0.0-00010101000000-000000000000 dependency. It's hard to trace down every module that needs it, so I applied replace for all module. It will be removed after this week's release anyway.

Reply:
go fails if just runs into `v0.0.0-00010101000000-000000000000` at any point
```
go mod tidy -compat=1.17
go: go.opentelemetry.io/collector@v0.48.1-0.20220412005140-8eb68f40028d requires
	go.opentelemetry.io/collector/pdata@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

Reply:
👍 it's fine to keep it if we are going to remove it soon (we just have to remember to add it too at the opentelemetry-collector-releases repo)